src/main/java/frc/robot/controllers/SparkMaxPositionController.java

```diff
@@ -164,7 +164,7 @@ public boolean isHomed() {
   	@Override
 	public void requestPosition(double requestedPosition) {
   	  if ( ! isPositionWithinSoftLimits(requestedPosition)) {
-        DriverStation.reportWarning("Invalid Position " + requestedPosition, false);
+        DriverStation.reportWarning("Invalid Position " + config.getName() ,false);
```

Comment:
We might still want to see the requested position. Suggest something like:

DriverStation.reportWarning("Invalid Position " + requestedPosition + " for " + config.getName() ,false);